searx/infopage/en/privacy-policy.md

```diff
@@ -0,0 +1,217 @@
+# Privacy Policy
+
+We provide this information for SearXNG's instances installed on servers in the
+Economic European Area (EEA) or outside the EEA for those who consult the
+{{link('SearXNG website', 'search')}} or submit queries.  So that you know, this
+information applies only to this SearXNG instance **and not to other websites
+the user may consult through links**.
+
+When data subjects or SearXNG providers are in the EEA, the [EU Regulation
+2016/679 (GDPR)](https://eur-lex.europa.eu/legal-content/EN/TXT/PDF/?uri=CELEX:32016R0679&from=EN)
+applies.
+
+Indeed, according to Article 3(2) of the GDPR, we underline whether SearXNG's
+instances are installed on servers, not in the EEA, but the service is offered
+to users in the EEA, Regulation 2016/679 applies.
+
+> Article 3
+> Territorial scope
+>
+> 1. This Regulation applies to the processing of personal data in the context
+>    of the activities of an establishment of a controller or a processor in the
+>    Union, regardless of whether the processing takes place in the Union or
+>    not.
+>
+> 2. This Regulation applies to the processing of personal data of data subjects
+>    who are in the Union by a controller or processor not established in the
+>    Union, where the processing activities are related to:
+>
+>    (a) the offering of goods or services, irrespective of whether a payment of
+>        the data subject is required, to such data subjects in the Union; or
+>
+>    (b) the monitoring of their behaviour as far as their behaviour takes place
+>        within the Union.
+>
+> 3. This Regulation applies to the processing of personal data by a controller
+>    not established in the Union, but in a place where Member State law applies
+>    by virtue of public international law.
+
+## Data controller
+
+The data controller is `{name of the instance maintaner}` - (`{maintainer's
+email}`).
```

Comment:
`{name of the instance maintaner}`:   We do not have the name of the instance's maintainer in the config.

`{maintainer's email}`: We have only one (optional) contact URL in the settings. This URL can be a `mailto:` URL to send a mail, but this URL can also link to a contact form. 

https://github.com/searxng/searxng/blob/52f6bc745bfff4f0cb523098fae624656e54c82e/searx/settings.yml#L11-L12

Reply:
From my perspective, the name of the instance maintainer should be present only in the full text of the privacy policy.

FYI and for all, according to article 13(1((a) the privacy policy should contain:
> (a) the identity and the contact details of the controller and, where applicable, of the controller's representative;

Article 4(1)(7) of the GDPR define the controller as follows:

> *‘controller’ means the natural or legal person, public authority, agency or other body which, alone or jointly with others, determines the purposes and means of the processing of personal data; where the purposes and means of such processing are determined by Union or Member State law, the controller or the specific criteria for its nomination may be provided for by Union or Member State law*

Ultimately, we should leave the responsibility to the maintainer of the SearXNG instance to put his or her name.